pyanaconda/modules/common/errors/subscription.py

```diff
@@ -37,3 +37,9 @@ class UnregistrationError(AnacondaError):
 class SubscriptionError(AnacondaError):
     """Subscription attempt failed."""
     pass
+
+
+@dbus_error("KickstartRegistrationError", namespace=ANACONDA_NAMESPACE)
+class KickstartRegistrationError(AnacondaError):
```

Comment:
Please, move this error to `pyanaconda.ui.lib.subscription` as well. This exception is not raised by a DBus module, it is just an internal workaround for the UI. Anyway, DBus modules don't raise kickstart-related exceptions. Modules return a kickstart report during parsing a kickstart file and raise more generic errors after that.

Reply:
Moved as well. BTW, I should keep the `@dbus_error` stuff in place, right ?

Reply:
No, you should remove it since it is not a DBus error.